f34-py39/s2i_assemble.patch

```diff
@@ -83,6 +86,11 @@
 -  pip install -r requirements.txt
 -fi
 +thamos install
-
+ 
  if [[ -f setup.py && -z "$DISABLE_SETUP_PY_PROCESSING" ]]; then
    echo "---> Installing application ..."
+-  pip install .
++  pip install --no-cache-dir .
```

Comment:
We could eventually also include ``--disable-pip-version-check``. Here and in other ``install`` parts above:

```suggestion
+  pip install --no-cache-dir --disable-pip-version-check .
```

By the way, pip also supports passing options as environment variables:

https://pip.pypa.io/en/stable/topics/configuration/#environment-variables

We could set all the options globally and all the pip calls would inherit this configuration in the container image.

Reply:
> @fridex , any counter-arguments?

Out of bullets 🔫  👍🏻